README.md

```diff
@@ -42,19 +42,42 @@ Future features include managing/creating email campaigns and ticket exchanges.
 3. Create mkcert certificates.
    1. Navigate to `<path/to/WonderTix/server>` and `<path/to/WonderTix/client>`.
    2. Run `mkcert -install` to install the local certificate authority.
-   3. Run `mkcert localhost` to create a certificate.   
+   3. Run `mkcert localhost` to create a certificate.
+   4. Node does not automatically look in the system root store for certificate authority files, nor would Docker include it anyways. Thus, the certificate authority generated by mkcert must be copied to a location where the server can find it so features like WebSockets function. To do so, in the server directory, run the following command:
+      - On Windows Powershell: `copy ($(mkcert -CAROOT) + '/rootCA.pem') .`.
+      - On Mac/Linux: `cp $(mkcert -CAROOT)/rootCA.pem .`.
 4. Run `docker-compose up -d` (or `docker compose up` to see live container logs in the terminal). This starts a locally running instance of Wondertix using docker containers. 
 5. To test the checkout process with Stripe, make sure you have the [Stripe CLI](https://stripe.com/docs/stripe-cli) installed. 
    1. Once the Stripe CLI is installed, run `stripe login` and press enter to open the browser. 
    2. In the browser that the Stripe CLI opens, make sure you select `Wondertix Dev` as the account, verify that the code matches that in your terminal, and select `Allow Access`. If `Wondertix Dev` does not appear as an option, you will need to ask for access to the account. You should only need to login to the account like this once. 
-   3. Back in your terminal, run `stripe listen --forward-to https://localhost:8000/api/2/order/webhook --events checkout.session.completed,checkout.session.expired` and copy the resulting ***signing secret*** as your `PRIVATE_STRIPE_WEBHOOK` variable in the `.env` file. This tells Stripe to forward events to your local webhook. See the `stripe listen` command [API docs](https://stripe.com/docs/cli/listen) for more details. 
+   3. Back in your terminal, run `stripe listen --forward-to https://localhost:8000/api/2/order/webhook` and copy the resulting ***signing secret*** as your `PRIVATE_STRIPE_WEBHOOK` variable in the `.env` file. This tells Stripe to forward events to your local webhook. See the `stripe listen` command [API docs](https://stripe.com/docs/cli/listen) for more details. 
```

Comment:
@benfaustin95 Do you know if changing this command if going to affect order processing?

Reply:
The `--events` parameter is somewhat of a filter for what events pass through to the webhook, for the card reader it is required that payment_intent and terminal.reader events are passed to the webhook, which is the reason this parameter was removed. Since checkout sessions carry `__ticketing` metadata, and those are the only events passed through the ticketing webhook, and based on testing, I believe this lack of filtering should not effect the order processing.

Reply:
Looks good to me